apps/web/lib/api/tools.ts

```diff
@@ -6,24 +6,33 @@ const PROJECTS_API_BASE_URL = new URL(
   process.env.NEXT_PUBLIC_API_URL,
 ).href;
 
-export const SendJoinFeatureRequest = async (body: PostJoin) => {
-  await fetchWithAuth(
-    `${PROJECTS_API_BASE_URL}/join`,
-    {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify(body),
-    },
-  );
-}
+export const SendJoinFeatureRequest = async (
+  body: PostJoin,
+  projectId: string,
+) => {
+  await fetchWithAuth(`${PROJECTS_API_BASE_URL}/join?project_id=${projectId}`, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(body),
+  })
+    .then((res) => res.json())
+    .then((data) => console.log(data))
+    .catch((err) => console.log(err));
+};
```

Comment:
It would be better to have this on lowerCase first letter. (e.g sendJoinFeatureReqeust) to keep it consistent with others .